lib/checks/label/multiple-label.js

```diff
@@ -26,4 +19,16 @@ while (parent) {
 }
 
 this.relatedNodes(labels);
+
+if (labels.length > 1) {
+	const visibleLabels = labels.filter(label =>
+		axe.commons.dom.isVisible(label, true)
+	);
+	if (visibleLabels.length > 1) {
+		return true;
+	}
+	const labelledby = axe.commons.dom.idrefs(node, 'aria-labelledby');
+	return !labelledby.includes(visibleLabels[0]);
+}
+
 return labels.length > 1;
```

Comment:
This could be `returns false` since the `true` case is handled above

Reply:
But then it would then pass even if the `aria-labelledby` does not include the correct idRef. I'll update with some comments.

Reply:
Good point. I've also added some comments and changed a variable name to make it a little more clear.